pom.xml

```
@@ -1,40 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.jenkins-ci.plugins</groupId>
-    <artifactId>plugin</artifactId>
-    <version>4.40</version>
-    <relativePath />
+    <groupId>org.jenkins-ci</groupId>
+    <artifactId>jenkins</artifactId>
+    <version>1.78</version>
   </parent>
 
-  <artifactId>pipeline-metadata-utils</artifactId>
   <groupId>org.jenkins-ci.infra</groupId>
+  <artifactId>pipeline-metadata-utils</artifactId>
   <version>1.3-SNAPSHOT</version>
   <name>Pipeline Metadata Utils</name>
   <description>Utilities for use in generating metadata and documentation for Pipeline steps or other Describables</description>
 
   <properties>
-    <jenkins.version>2.319.3</jenkins.version>
+    <gitHubRepo>jenkins-infra/pipeline-metadata-utils</gitHubRepo>
+    <spotbugs.excludeFilterFile>${project.basedir}/src/spotbugs/spotbugs-excludes.xml</spotbugs.excludeFilterFile>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
-	<groupId>io.jenkins.tools.bom</groupId>
-	<artifactId>bom-2.319.x</artifactId>
-	<version>1466.v85a_616ea_b_87c</version>
-	<scope>import</scope>
-	<type>pom</type>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-weekly</artifactId>
+        <version>1466.v85a_616ea_b_87c</version>
+        <type>pom</type>
+        <scope>import</scope>
       </dependency>
     </dependencies>
   </dependencyManagement>
 
   <dependencies>
+    <dependency>
+      <groupId>org.jenkins-ci.main</groupId>
+      <artifactId>jenkins-core</artifactId>
+      <version>2.355</version>
```

Seems like we should depend on the most recent weekly release in this tool, just as we depend on the most recent weekly release in the steps generator.
```suggestion
      <version>2.357</version>
```

See https://github.com/jenkins-infra/pipeline-steps-doc-generator/pull/183 and https://github.com/jenkins-infra/pipeline-steps-doc-generator/pull/182

Should configure dependabot in a future pull request so that this library keeps pace with weekly.